go/cmd/deploy/control_plane.go

```diff
@@ -178,12 +178,13 @@ func (c *ControlPlaneClient) processNewSteps(
 				Step:      step,
 				Status:    currentStatus,
 			}
-
 			if err := onStepUpdate(event); err != nil {
 				return err
 			}
-		}
 
+			// INFO: This is for demo purposes only.
+			time.Sleep(800 * time.Millisecond)
```

Comment:
what does it demonstrate?

Reply:
Sorry for not being descriptive. Since we now work locally everything happens so fast, thanks to that sleep we can clearly see each part in control plane, like booting up the vm, running rootfs etc... I'll add more descriptive comment  this is mainly for demo so we can show them whats happening when you invoke the deploy